core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -75,6 +75,21 @@ private MetadataColumns() {}
   public static final String DELETE_FILE_ROW_FIELD_NAME = "row";
   public static final int DELETE_FILE_ROW_FIELD_ID = Integer.MAX_VALUE - 103;
   public static final String DELETE_FILE_ROW_DOC = "Deleted row values";
+  public static final NestedField CHANGELOG_OPERATION =
```

Comment:
I wonder whether we should add `ReservedColumns` as a separate class. There are a few columns that are not metadata columns as such.

Reply:
At this point, `MetadataColumns` references both metadata columns with IDs from `Integer.MAX_VALUE - (1-100)` and reserved columns with IDs `Integer.MAX_VALUE - (101-200)`. To me, changelog-related columns are more like reserved columns as it is not something we can request on demand. That's why I did not add underscore in their names.

Reply:
Kept as `MetadataColumns` for now. Resolving as no change.